Index: src/main/java/org/artofsolving/jodconverter/office/ManagedOfficeProcess.java
===================================================================
--- src/main/java/org/artofsolving/jodconverter/office/ManagedOfficeProcess.java	(Revision 205)
+++ src/main/java/org/artofsolving/jodconverter/office/ManagedOfficeProcess.java	(Arbeitskopie)
@@ -174,5 +174,10 @@
             throw new OfficeException("could not terminate process", exception);
         }
     }
+    
+    boolean isConnected()
+    {
+    	return connection.isConnected();
+    }
 
 }
Index: src/main/java/org/artofsolving/jodconverter/office/ProcessPoolOfficeManager.java
===================================================================
--- src/main/java/org/artofsolving/jodconverter/office/ProcessPoolOfficeManager.java	(Revision 205)
+++ src/main/java/org/artofsolving/jodconverter/office/ProcessPoolOfficeManager.java	(Arbeitskopie)
@@ -109,4 +109,8 @@
         }
     }
 
+	public boolean isRunning() {
+		return running;
+	}
+
 }
Index: src/main/java/org/artofsolving/jodconverter/office/OfficeManager.java
===================================================================
--- src/main/java/org/artofsolving/jodconverter/office/OfficeManager.java	(Revision 205)
+++ src/main/java/org/artofsolving/jodconverter/office/OfficeManager.java	(Arbeitskopie)
@@ -32,4 +32,5 @@
 
     void stop() throws OfficeException;
 
+    boolean isRunning();
 }
Index: src/main/java/org/artofsolving/jodconverter/office/ExternalOfficeManager.java
===================================================================
--- src/main/java/org/artofsolving/jodconverter/office/ExternalOfficeManager.java	(Revision 205)
+++ src/main/java/org/artofsolving/jodconverter/office/ExternalOfficeManager.java	(Arbeitskopie)
@@ -89,4 +89,8 @@
         }
     }
 
+	public boolean isRunning() {
+		return connection.isConnected();
+	}
+
 }
Index: src/main/java/org/artofsolving/jodconverter/office/PooledOfficeManager.java
===================================================================
--- src/main/java/org/artofsolving/jodconverter/office/PooledOfficeManager.java	(Revision 205)
+++ src/main/java/org/artofsolving/jodconverter/office/PooledOfficeManager.java	(Arbeitskopie)
@@ -108,4 +108,8 @@
         managedOfficeProcess.stopAndWait();
     }
 
+	public boolean isRunning() {
+		return managedOfficeProcess.isConnected();
+	}
+
 }
